templates/base/delete_modal_actions.tmpl

```diff
@@ -1,10 +1,10 @@
 <div class="actions">
-	<div class="ui red basic inverted cancel button">
+	<button type="button"{{/* To prevent accidental submission when clicking on no */}} class="ui red basic inverted cancel button">
```

Comment:
Is it handled by JS?  If yes, then no need "type=button"?

Reply:
At the moment, it isn't.
An easy workaround we could do is `document.querySelectorAll('.cancel').forEach(cancelButton => cancelButton.setAttribute('type', 'button'))`.
That should always work, but it would set it dynamically instead of hardcoded.
It depends on what we want:
- A site that works (mostly) correctly even when JS is disabled
- A site with as minimal code as possible
Both approaches have advantages and disadvantages.

Reply:
Last question:
Do we keep the already existing `type="button"`s in, or should I remove them again?

Reply:
If we decided to use JS and make JS correct, then I think we can remove `type="button"` to keep clear? Either is fine to me.

Reply:
Good, then I'm removing the attributes agian now.